bin/serve.js

```diff
@@ -59,6 +74,34 @@ var app = jserve({
 // Start the JServe application
 app.start();
 
+// Middleware to serve JSON at the index route
+// For eg. A request to / will return data/index.json
+function interceptIndex(request, response, next) {
```

Comment:
I had a chat with one of the devs here, and we think it might be cleaner to modify `request.path` and let the data flow through the `jserve` in the usual fashion, and let it handle reading from disk etc. So how about;

```
// Middleware to serve JSON at the index route
// e.g. a request to / will return data/index.json
function interceptIndex(request, response, next) {
	if (args.index === true && request.path === '/') {
		request.path = '/index.json';
	}
	return next();
}
```

(also note slight modification to comment 😉 )